build-tools/packages/build-tools/src/fluidBuild/fluidUtils.ts

```diff
@@ -159,18 +159,21 @@ let defaultConfigLoaded = false;
  */
 export function getFluidBuildConfig(
 	searchDir: string,
+	warnNotFound = true,
```

Comment:
update docs to reflect this parameter (can't make a suggestion). potentially also name `warnOnNotFound`?

suggested docs:

```typescript
 * @param warnNotFound - Whether to warn if no fluidBuild config is found.
```

Reply:
Done.